adapter/internal/oasparser/envoyconf/http_filters.go

```diff
@@ -167,6 +167,14 @@ func getRateLimitFilter() *hcmv3.HttpFilter {
 				},
 			},
 		},
+		ResponseHeadersToAdd: []*corev3.HeaderValueOption{
+			{
+				Header: &corev3.HeaderValue{
+					Key:   xRateLimitEnforcedHeader,
+					Value: xRateLimitEnforcedValue,
+				},
+			},
+		},
```

Comment:
Let's remove this header as we do not need these any more. Since we can use the "RL" flag.